src/app/communities/page.tsx

```diff
@@ -16,16 +17,21 @@ export default function Communities() {
         <Image
           src="/images/communities-banner.svg"
           alt="Communities"
-          width={0}
-          height={0}
-          style={{ width: '100%', height: 'auto' }}
+          width={2272}
+          height={378}
+          className="w-full h-auto mb-[60px] mt-[60px]"
         />
       </div>
-      <div className="flex flex-wrap mt-[16px] gap-[24px]">
-        {communitiesToRender.map(community => (
-          <CommunityItem key={community.title} {...community} />
-        ))}
-      </div>
+      {Object.entries(communitiesByCategory).map(([category, communities]) => (
+        <div key={category} className="mb-[54px]">
+          <HeaderTitle>{category}</HeaderTitle>
```

Comment:
This title has 32px instead of 24px on design.

Reply:
THE BEST! GREAT CATCH!